docs/source/technical_tutorials/access/copc_access.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>It looks like you created this notebook based upon this one: <a href="https://guide.cloudnativegeo.org/copc/lidar-las-to-copc.html" target="_blank">https://guide.cloudnativegeo.org/copc/lidar-las-to-copc.html</a></p><p> </p><p>However, this notebook appears to be missing several steps and uses maap-py, but describes earthaccess instead.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/459/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>